cypress/e2e/draw/draw-feat-style.cy.ts

```diff
@@ -0,0 +1,335 @@
+          ).should('be.lessThan', 0.0001)
+        })
+    })
+  })
+})
+
+describe('Test style edition of Line feature', () => {
+  beforeEach(() => {
+    cy.visit('/')
+    describe('Draw "Line"', () => {
+      cy.get('button[data-cy="drawButton"]').click()
+      cy.get('button[data-cy="drawLineButton"]').click()
+      cy.get('div.ol-viewport').click(200, 400, { force: true })
+      cy.get('div.ol-viewport').click(300, 300, { force: true })
+      cy.get('div.ol-viewport').click(400, 400, { force: true })
+      cy.get('div.ol-viewport').dblclick(500, 300, { force: true })
+    })
+    describe('Edit feature style', () => {
+      cy.get('*[data-cy="featItemActionStyle"]').click()
+      cy.get('*[data-cy="featStyleColor"]')
+        .invoke('val', '#0088ff')
+        .trigger('input')
+      cy.get('*[data-cy="featStyleLineWidth"]')
+        .find('input')
+        .eq(1)
+        .type('{selectAll}7.5', { force: true })
+      cy.get('*[data-cy="featStyleLineStyle"]').find('button').eq(1).click()
+      cy.get('button[data-cy="featureEditValidate"]').click()
+    })
+  })
+
+  describe('When checking featureStyle in OL DrawnFeature object', () => {
+    it('feature style shall be applied', () => {
+      // get featureLayer
+      cy.window()
+        .its('olMap')
+        .then(function (olMap) {
+          const featureLayers = olMap
+            .getLayers()
+            .getArray()
+            .filter((l: any) => l.get('cyLayerType') == 'featureLayer')
+          const features = featureLayers
+            .map((l: any) => l.getSource().getFeatures())
+            .flat()
+          cy.wrap(features.length).should('equal', 1)
+          const ff = features[0]
+          cy.wrap(ff.featureType).should('equal', 'drawnLine')
+          // check color blue
+          cy.wrap(ff.featureStyle.color).should('equal', '#0088ff')
+          // check line width 5
+          cy.wrap(ff.featureStyle.stroke).should('equal', 7.5)
+          // check line style dashed
+          cy.wrap(ff.featureStyle.linestyle).should('equal', 'dashed')
+        })
+    })
+  })
+})
+
+describe('Test style edition of Polygon feature', () => {
+  beforeEach(() => {
+    cy.visit('/')
+    describe('Draw "Polygon"', () => {
+      cy.get('button[data-cy="drawButton"]').click()
+      cy.get('button[data-cy="drawPolygonButton"]').click()
+      cy.get('div.ol-viewport').click(100, 100, { force: true })
+      cy.get('div.ol-viewport').click(100, 200, { force: true })
+      cy.get('div.ol-viewport').dblclick(200, 200, { force: true })
+    })
+    describe('Edit feature style', () => {
+      cy.get('*[data-cy="featItemActionStyle"]').click()
+      cy.get('*[data-cy="featStyleColor"]')
+        .invoke('val', '#0000ff')
+        .trigger('input')
+      cy.get('*[data-cy="featStyleLineWidth"]').type('{selectAll}5.5')
+      cy.get('*[data-cy="featStyleTransparency"]').type('{selectAll}22')
+      cy.get('button[data-cy="featureEditValidate"]').click()
```

Comment:
why do you wrap all this in a describe statement?

Reply:
I'll have to check potential reorganisation